### DISCUSSION TITLE
Dont know if this is my fuckup or something but

### DISCUSSION THREAD

**Discussion by atari-monk** [bilal-fazlani/commanddotnet#475] [General]
![image](https://user-images.githubusercontent.com/5488398/219667891-0f0499d8-ab17-4e64-979a-c66ab87dc40d.png)
I want to update to latest version of commanddotnet
i will remove repl version
i clean my code
found some old example from commanddotnet docs
this fancy bdd style test dosent work after i updated nugets
unless i dont see something.

**Comment by drewburlingame** [comment]
It's hard to know without seeing the app.  It looks like the value `60` might be written twice.   Is this repo publicly available?   It's hard to know what's going on without seeing the app and the tests.  What version are you upgrading from and to?

**Reply by atari-monk** [reply]
https://github.com/atari-monk/net-my-libs
Delete all of it exept  independent proj CommandDotNet.App and CommandDotNet.App.Tests
i commented broken test.
Open them in VSCode as workspace, rest will be isolated
from CommandDotNet" Version="6.0.5"
to "CommandDotNet" Version="7.0.2"
